### DISCUSSION TITLE
QGEP plugins: Inclusion of QGIS Password manager

### DISCUSSION THREAD

**Discussion by cymed** [QGEP/QGEP#793] [Ideas]
At the moment, all QGEP plugins are based on complete pg_service.conf. This means that all passwords are stored in clear text somewhere, which poses a security threat, especially if the passwords on postgres are linked to other applications. 
QGIS has its own functionality to save authentication configurations (authcfg). It would be nice to be able to use the authcfg in our plugins. 

@ technical group: is it viable to access this authentication version?

**Comment by sjib** [comment]
Some background:
- Olivier did some work to adjust the handling of pg_conf: https://github.com/QGEP/qgepqwat2ili/pull/97
- The issue was discussed in various forms in the [QGIS Modelbaker project](https://github.com/opengisch/QgisModelBaker/issues?q=is%3Aissue+password
- And also in ili2db: https://github.com/claeis/ili2db/issues/448 - so as we use ili2db this would needed to be implemented there.

**Reply by cymed** [reply]
Is the datamodel tool also linked to ili2db?

**Reply by sjib** [reply]
I don't think so: https://github.com/QGEP/qgepplugin/blob/master/qgepplugin/gui/qgepdatamodeldialog.py
